### PR TITLE
Hub 410, 423, 424 New content notifications

### PR DESCRIPTION
**Pull request by swilson-mojhub**

### CODE REVIEW

File: moj-fe/app/Http/Controllers/HubLinksController.php [hunk 1]
```diff
@@ -15,11 +16,13 @@ function getItem(Request $request, $id = NULL)
...
 		$path = LangSelectPath::getPath($request->path());
 		$backlink = HubBackLink::getBackLink();
+		$new_content = HubLinks::checkNewContent($request->input('user_id'));
 
 		return view('hub.item', [
 			'page' => $page_data,
 			'path' => $path,
-			'backlink' => $backlink
+			'backlink' => $backlink,
+            'newcontent' => $new_content
```
Comment (lazzurs) [inline moj-fe/app/Http/Controllers/HubLinksController.php]: Fix identing please.

File: moj-fe/app/Repositories/HubLinksRepository.php [hunk 2]
```diff
@@ -37,4 +37,18 @@ public function getItem($id = NULL, $user_id = NULL) {
...
+    public function checkNewContent($user_id = NULL) {
+        $url = $this->locale . '/api/checknewcontent';
+
+        $headers = [];
+
+        if ($user_id) {
+            $headers['custom-auth-id'] = $user_id;
+        }
+
+        $response	= $this->client->get($url, [ 'headers' => $headers ]);
```
Comment (lazzurs) [inline moj-fe/app/Repositories/HubLinksRepository.php]: Please fix whitespace.

File: moj-fe/app/Repositories/NewContentRepository.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+use GuzzleHttp\Exception\ClientException;
+
+class NewContentRepository {
+    protected $client;
+    protected $locale = '';
+
+    public function __construct() {
+        $this->client = new Client(array(
+            'base_uri' => config('app.api_uri'),
+            'timeout' => 60.0
```
Comment (lazzurs) [inline moj-fe/app/Repositories/NewContentRepository.php]: Is this not a configuration value somewhere and if not could we change it to one please?

File: moj-fe/config/app.php [hunk 4]
```diff
@@ -159,6 +159,7 @@
         Mcamara\LaravelLocalization\LaravelLocalizationServiceProvider::class,
 
         App\Providers\VideosRepositoryServiceProvider::class,
+        App\Providers\NewContentRepositoryServiceProvider::class,
 		App\Providers\HubLinksRepositoryServiceProvider::class,
 		App\Providers\PdfsRepositoryServiceProvider::class,
         App\Providers\NewsRepositoryServiceProvider::class,
```
Comment (lazzurs) [inline moj-fe/config/app.php]: Please fix indentation.

File: moj-be/core/phpunit.xml.dist [hunk 5]
```diff
@@ -24,9 +24,9 @@
     <!-- Do not limit the amount of memory tests take to run. -->
     <ini name="memory_limit" value="-1"/>
     <!-- Example SIMPLETEST_BASE_URL value: http://localhost -->
-    <env name="SIMPLETEST_BASE_URL" value=""/>
+    <env name="SIMPLETEST_BASE_URL" value="http://localhost:8182"/>
     <!-- Example SIMPLETEST_DB value: mysql://username:password@localhost/databasename#table_prefix -->
-    <env name="SIMPLETEST_DB" value=""/>
+    <env name="SIMPLETEST_DB" value="mysql://hubdb_user:hubdb_pass@localhost/hubdb"/>
```
Comment (swilson-mojhub) [inline moj-be/core/phpunit.xml.dist]: Hmmm, I think they are... is this file not being used as s fallback?

**Pull request merged**